### PR TITLE
Add post about 1aim inviting Working Groups

### PR DESCRIPTION
**Pull request by Hoverbear**
We're looking at inviting some folks from the working groups to RustFest. :)

Please **do not merge** until Friday as I'm waiting for a response from a thread.

### CODE REVIEW

File: _posts/2018-04-19-working-group-invitation.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## What is a Working Group?
+
+[Working groups](https://internals.rust-lang.org/t/announcing-the-2018-domain-working-groups/6737) are assemblies of contributors. Working Groups work directly with the Core Team and focus on the imporving the end-to-end user experience of using Rust in their respecitve domain. That work can involve the language, compiler, libraries, tools, documentation, discoverability, and more! The overall effort is organized by Aaron Turon (`aturon`).
+
+* Networking services, lead by `withoutboats` and `cramertj`
+* [WebAssembly](https://internals.rust-lang.org/t/come-join-the-rust-and-webassembly-working-group/6845), lead by `fitzgen`
+* [CLI apps](https://internals.rust-lang.org/t/announcing-the-cli-working-group/6872), lead by `killercup`
+* [Embedded devices](https://internals.rust-lang.org/t/announcing-the-embedded-devices-working-group/6839), lead by `japaric`
```
Comment (badboy) [inline _posts/2018-04-19-working-group-invitation.md]: Is this a complete list? Do we need to make it more clear if it is not?

File: _posts/2018-04-19-working-group-invitation.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    * In addition to those that RustFest & Diversity Sponsor Donations support
+    * Selected from previous **[Diversitytickets.org event](https://diversitytickets.org/events/152)**.
+
+**What do you get?**
+* One RustFest Paris **ticket**.
+* **Hotel accommodation** from Sept. 29 to Oct. 2. (Provided by 1aim)
+* Up to **150€** for EU and **250€** for non-EU participants toward your travel expenses (reimbursed post-conference, pending receipts)
+
+**How do you participate?**
+If are nominated by a WG lead, **sign up [here](https://goo.gl/forms/A1hrLjA81kiBidff2)** until the **29th of April 2018**. If there are more than 12 nominated applications, the winners will be randomly drawn from the pool. You will have one week to accept your invitation or it will go to someone else.
```
Comment (badboy) [inline _posts/2018-04-19-working-group-invitation.md]: > If **you** are nominated

Comment (badboy) [reply _posts/2018-04-19-working-group-invitation.md]: Is 29th enough time? It's already quite late and I'd like to see this done quickly, but do we need to give people more time?

**Review by badboy — DISMISSED** [review]

File: _posts/2018-04-19-working-group-invitation.md (hunk 1 above)
Comment (Hoverbear) [reply _posts/2018-04-19-working-group-invitation.md]: It's not. But these are the teams I understand to be participating.

File: _posts/2018-04-19-working-group-invitation.md (hunk 2 above)
Comment (Hoverbear) [reply _posts/2018-04-19-working-group-invitation.md]: I don't think it's enough time, but I think most of the people who would qualify have already been participating.

**Comment by Hoverbear** [comment]
@yannleretaille Please take a look at this feedback. :)

File: _posts/2018-04-19-working-group-invitation.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+* Networking services, lead by `withoutboats` and `cramertj`
+* [WebAssembly](https://internals.rust-lang.org/t/come-join-the-rust-and-webassembly-working-group/6845), lead by `fitzgen`
+* [CLI apps](https://internals.rust-lang.org/t/announcing-the-cli-working-group/6872), lead by `killercup`
+* [Embedded devices](https://internals.rust-lang.org/t/announcing-the-embedded-devices-working-group/6839), lead by `japaric`
+
+We have also seen other groups forming such as the [Formal Verification](https://internals.rust-lang.org/t/announcing-the-formal-verification-working-group/7240), [Traits](https://internals.rust-lang.org/t/announcing-traits-working-group/6804) and [Rust Compiler Performance](https://internals.rust-lang.org/t/rust-compiler-performance-working-group/6934) working groups. Maybe we'll even see more form!
+
+You can also contact your **lead** or **Yann from 1aim (`Yann` on irc.mozilla)** directly with questions.
```
Comment (dns2utf8) [inline _posts/2018-04-19-working-group-invitation.md]: I would include the "your" with the **lead**

File: _posts/2018-04-19-working-group-invitation.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+**How do you participate?**
+If are nominated by a WG lead, **sign up [here](https://goo.gl/forms/A1hrLjA81kiBidff2)** until the **29th of April 2018**. If there are more than 12 nominated applications, the winners will be randomly drawn from the pool. You will have one week to accept your invitation or it will go to someone else.
+
+
+Again, big thanks to **1aim** and we’re excited to see them (and **you!**) at **RustFest** this year!
+
+![1aim Logo](https://blog.rustfest.eu/assets/sponsors/1aim.svg)
+
+**A little bit about our friends over at 1aim:**
```
Comment (dns2utf8) [inline _posts/2018-04-19-working-group-invitation.md]: Maybe turn this into a h2

**Review by dns2utf8 — COMMENTED** [review]
I have only a small nit. Lgtm

**Comment by yannleretaille** [comment]
@hoverbear im OK with releasing this

**Pull request merged**